src/stcal/ramp_fitting/ols_cas22/_fixed.pyx

```diff
@@ -0,0 +1,187 @@
+"""
```

Comment:
I need to remove the use of negative indices here so I can remove some types of checks so that its faster. This is relatively simple.